### PR TITLE
OBW: Remove live rates from shipping step, promote label printing.

### PR DESCRIPTION
**Pull request by jeffstieler**
This PR seeks to:
* Remove the WCS-powered Live Rates shipping method from the OBW
* Add a mention about where to find live rate extensions
* Recommend WCS-powered Shipping Labels to stores in the US
* Recommend ShipStation (for labels) to stores in Australia, Canada, and the UK
* Tweak the weight/dimension unit selection to be inline in a sentence

Screenshot from US store:
screen shot 2018-08-30 at 3 20 21 pm

Screenshot from UK store:
screen shot 2018-08-30 at 3 22 55 pm

Screenshot from a store with no recommended fulfillment option:
screen shot 2018-08-30 at 3 21 22 pm

cc: @jameskoster @aleftick 

_See p6riRB-3uE-p2 for full context._

### CODE REVIEW

File: includes/admin/class-wc-admin-setup-wizard.php [hunk 1]
```diff
@@ -143,6 +144,57 @@ protected function should_show_recommended_step() {
 			|| $this->should_show_mailchimp();
 	}
 
+	public function enqueue_scripts() {
```
Comment (claudiulodro) [inline includes/admin/class-wc-admin-setup-wizard.php]: Missing docblock

File: includes/admin/class-wc-admin-setup-wizard.php [hunk 2]
```diff
@@ -871,31 +890,25 @@ public function wc_setup_shipping() {
...
 		}
 
+		$is_wcs_labels_supported  = $this->is_wcs_shipping_labels_supported_country( $country_code );
+		$is_shipstation_supported = $this->is_shipstation_supported_country( $country_code );
+
 		?>
 		<h1><?php esc_html_e( 'Shipping', 'woocommerce' ); ?></h1>
+		<?php if ( $intro_text ) : ?>
 		<p><?php echo wp_kses_post( $intro_text ); ?></p>
+		<?php endif; ?>
```
Comment (claudiulodro) [inline includes/admin/class-wc-admin-setup-wizard.php]: `if` block needs some indenting inside it.

File: includes/admin/class-wc-admin-setup-wizard.php [hunk 3]
```diff
@@ -1016,20 +1047,29 @@ public function wc_setup_shipping_save() {
...
+		if ( $setup_wcs_labels ) {
+			$this->install_woocommerce_services();
+		}
+
+		if ( $setup_shipstation ) {
+			$this->install_plugin(
+				'woocommerce-shipstation',
+				array(
+					'name'      => __( 'ShipStation', 'woocommerce' ),
+					'repo-slug' => 'woocommerce-shipstation',
```
Comment (claudiulodro) [inline includes/admin/class-wc-admin-setup-wizard.php]: This doesn't seem to be a valid slug on .org, and the plugin doesn't install when I try to install it from the wizard. Is it not live on .org yet? If not, when will it be?

File: includes/admin/class-wc-admin-setup-wizard.php (hunk 1 above)
Comment (claudiulodro) [reply includes/admin/class-wc-admin-setup-wizard.php]: A handful of other [code style issues the linter picked up, also](https://travis-ci.org/woocommerce/woocommerce/jobs/422778909#L921).

**Review by claudiulodro — CHANGES_REQUESTED** [review]
Overall this looks good. Just a few code style issues and some questions regarding Shipstation. 

Will the Wizard-installed trial of Shipstation cause issues for users after the 30 days are over? How does the trial work when it's been installed from the setup wizard? e.g. Does the user go in and set everything up for Shipstation after the wizard is complete?

**Comment by jeffstieler** [comment]
> Will the Wizard-installed trial of Shipstation cause issues for users after the 30 days are over? How does the trial work when it's been installed from the setup wizard? e.g. Does the user go in and set everything up for Shipstation after the wizard is complete?

As it currently stands, there isn't any sort of automatic setup for ShipStation for users that install it from the OBW (e.g. there is no WCS tie-in). The 30 day trial is something offered by ShipStation themselves - users will need to go to ShipStation to setup an account. A new admin notice is being introduced to provide more guidance there - see p6riRB-3uE-p2.

File: includes/admin/class-wc-admin-setup-wizard.php (hunk 3 above)
Comment (jeffstieler) [reply includes/admin/class-wc-admin-setup-wizard.php]: It's not live on .org yet. Ideally we will have it up by the time the RC is cut - still need to work that out with @laurendavissmith.

**Review by claudiulodro — APPROVED** [review]
Thanks for clarifying. This looks good to merge.

**Pull request merged**